### DISCUSSION TITLE
Hold down modifier key to do Thirds and Half Thirds.

### DISCUSSION THREAD

**Discussion by AitoHikari** [mrkai77/Loop#477] [Q&A]
Hello, It would be great to have the ability to use a modifier key (Command, Control, Option) to open up more window management options, such as left third, right third, top left third, etc. This would allow for more flexible use of screen space beyond just quarters (left, right, top, bottom).

So using a key such as a modifier keys (or a specific hotkey) to open up more window management such as thirds and half thirds.

**Comment by SiddharthM18** [comment]
Hundred percent agree with this. As someone with a wider monitor and general needs for thirds anyways, this would be so incredibly useful.

**Comment by malcojus** [comment]
+1 for this! Ultrawide monitor users will greatly benefit from this feature

**Comment by plyght** [comment]
This would be awesome! or even some implementation sort of how [Lasso](https://thelasso.app) does it, with the screen showing up and you choosing the part you want. But if not that, then a modifier key would honestly do the trick (probably even easier/better than my idea) and hopefully this gets added soon! +1

**Comment by mrkai77** [comment]
Moving to #702 :)
